packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.5.2
 
 * Updates minimum Flutter version to 3.0.
+* Added option to configure HTTP headers via `VideoPlayerController` to fix access to M3U8 files on Android.
```

Comment:
Adds

Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style